src/app/components/tablist/tablist.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+// https://react-spectrum.adobe.com/react-aria/useTabList.html
+import {useTabList, useTab, useTabPanel} from 'react-aria';
+import {Item, useTabListState} from 'react-stately';
+import type {Node} from '@react-types/shared/src/collections';
+import './tablist.scss';
+
+type TabsProps = Parameters<typeof useTabListState>[0] &
+    Parameters<typeof useTabList>[0];
+
+function Tabs(props: TabsProps) {
+    const state = useTabListState(props);
+    const ref = React.useRef(null);
+    const {tabListProps} = useTabList(props, state, ref);
+
+    return (
+        <div className={`tabs ${props.orientation || ''}`}>
```

Comment:
is there a reason you didn't use `react-aria-components` like the example? https://react-spectrum.adobe.com/react-aria/Tabs.html seems like that would do some of the wiring you have above for you?

Reply:
That was actually code from the tablist page at react-aria. Makes sense that it would be in a package.
It doesn't seem to work the same, so I'll have some figuring out to do.

Reply:
It looks like react-aria-components don't play nice with Preact.